doc/P4-DPDK-starting-guide/table_manager.c

```diff
@@ -0,0 +1,226 @@
+#include <stdio.h>
```

Comment:
create and move this to e2e-test/p4rt. Provide make file also as part of test being added rather than providing in guide